settings/LoanPolicyDetail.js

```diff
@@ -101,6 +102,16 @@ class LoanPolicyDetail extends React.Component {
     const exReqPerInterval = get(policy, ['loansPolicy', 'existingRequestsPeriod', 'intervalId']);
     const gracePeriodInterval = get(policy, ['loansPolicy', 'gracePeriod', 'intervalId']);
     const timeOffsetInterval = get(policy, ['loansPolicy', 'openingTimeOffset', 'intervalId']);
+    const isOpeningTimeOffsetVisible = () => {
+      const {
+        loansPolicy,
```

Comment:
Are you sure about this variable? As far I remember it should be shortTermLoan and selected item from there. In our case it is `BEGINNING_OF_THE_NEXT_OPEN_SERVICE_POINT_HOURS`

Reply:
Definitely it should be shortTermLoan, but openingTimeOffset field appears only when `BEGINNING_OF_THE_NEXT_OPEN_SERVICE_POINT_HOURS` is selected in shortTerms

Reply:
and what if tomorrow this value will be available for long term load too?

Reply:
got the point